packages-next/types/src/index.ts

```diff
@@ -54,6 +54,7 @@ export type KeystoneConfig = {
   db: {
     adapter: 'mongoose' | 'knex';
     url: string;
+    onConnect?: (keystone: Keystone) => any;
```

Comment:
`Keystone` refers to a different thing here which is not what the argument to onConnect is

Reply:
Oh, right... 🤦

Reply:
Can we maybe pick a different name for that thing then? Not for this PR obviously, just flagging it as a thing for the future.